python/examples/memory/base.py

```diff
@@ -0,0 +1,25 @@
+import asyncio
+
+from beeai_framework.backend.message import AssistantMessage, SystemMessage, UserMessage
+from beeai_framework.memory.unconstrained_memory import UnconstrainedMemory
+
+
+async def main() -> None:
+    memory = UnconstrainedMemory()
+
+    # Single Message
+    await memory.add(SystemMessage("You are a helpful assistant"))
+
+    # Multiple Messages
+    await memory.add_many([UserMessage("What can you do?"), AssistantMessage("Everything!")])
+
+    print(memory.is_empty())  # false
+    print(memory.messages)  # prints saved messages as objects
```

Comment:
I would remove this line, it's not in the TS version and printing objects is always ugly output, plus a more readable outlet is done on line 20 already
```suggestion
```